prov/efa/src/efa_strerror.c

```diff
@@ -101,6 +102,22 @@ static const char *efa_errno_str(enum efa_errno err)
 		: errno_str[err - FI_EFA_ERRNO_OFFSET];
 }
 
+static const char *efa_debug_str(enum efa_errno err)
+{
+	switch (err) {
```

Comment:
I forget, would it be appropriate to handle RNR here?

Reply:
Sorry I must be missing the background. What is the context here, i.e. handle RNR in what sense?